### ISSUE TITLE
add support for .NET 6.0 <TargetFrameworks>

### ISSUE THREAD

**Issue by Bertk** [CycloneDX/cyclonedx-dotnet#451]
Please add support for .NET 6.0. CycloneDx-dotnet tool is failing without installing other .NET runtimes e.g. NET 5.0.

```
Tool 'cyclonedx' (version '2.2.0') was restored. Available commands: dotnet-CycloneDX

VERBOSE: run CycloneDX: 'dotnet dotnet-CycloneDX --exclude-test-projects --exclude-dev -o SBOM -biop ./artifacts -imp cycloneDX-metadata.xml Common.sln'
It was not possible to find any compatible framework version
The framework 'Microsoft.NETCore.App', version '5.0.0' (x64) was not found.
  - The following frameworks were found:
      6.0.0 at [C:\hostedtoolcache\windows\dotnet\shared\Microsoft.NETCore.App]

You can resolve the problem by installing the specified framework and/or SDK.


```

**Comment by coderpatros** [comment]
Available in v2.3.0

**Comment by sherknil** [comment]
using Cyclonedx 2.3.0 version, still getting below error

dotnet CycloneDX G:\Agent013\_work\1603\s/Lifecare.Care.AccessRights.sln -j -o ./ -r
========================== Starting Command Output ===========================
"C:\Windows\system32\cmd.exe" /D /E:ON /V:OFF /S /C "CALL "G:\Agent013\_work\_temp\b6899666-9d0e-4851-b6d5-1d65f285a32c.cmd""
It was not possible to find any compatible framework version
The framework 'Microsoft.NETCore.App', version '5.0.0' (x64) was not found.
  - The following frameworks were found:
      2.2.2 at [G:\Agent013\_work\_tool\dotnet\shared\Microsoft.NETCore.App]
      2.2.7 at [G:\Agent013\_work\_tool\dotnet\shared\Microsoft.NETCore.App]
      3.1.1 at [G:\Agent013\_work\_tool\dotnet\shared\Microsoft.NETCore.App]
      3.1.3 at [G:\Agent013\_work\_tool\dotnet\shared\Microsoft.NETCore.App]
      3.1.5 at [G:\Agent013\_work\_tool\dotnet\shared\Microsoft.NETCore.App]
      3.1.10 at [G:\Agent013\_work\_tool\dotnet\shared\Microsoft.NETCore.App]
      3.1.11 at [G:\Agent013\_work\_tool\dotnet\shared\Microsoft.NETCore.App]
      3.1.12 at [G:\Agent013\_work\_tool\dotnet\shared\Microsoft.NETCore.App]
      3.1.13 at [G:\Agent013\_work\_tool\dotnet\shared\Microsoft.NETCore.App]
      3.1.14 at [G:\Agent013\_work\_tool\dotnet\shared\Microsoft.NETCore.App]
      3.1.18 at [G:\Agent013\_work\_tool\dotnet\shared\Microsoft.NETCore.App]
      3.1.19 at [G:\Agent013\_work\_tool\dotnet\shared\Microsoft.NETCore.App]
      3.1.20 at [G:\Agent013\_work\_tool\dotnet\shared\Microsoft.NETCore.App]
      3.1.22 at [G:\Agent013\_work\_tool\dotnet\shared\Microsoft.NETCore.App]
      3.1.24 at [G:\Agent013\_work\_tool\dotnet\shared\Microsoft.NETCore.App]
      6.0.0 at [G:\Agent013\_work\_tool\dotnet\shared\Microsoft.NETCore.App]
      6.0.1 at [G:\Agent013\_work\_tool\dotnet\shared\Microsoft.NETCore.App]
      6.0.4 at [G:\Agent013\_work\_tool\dotnet\shared\Microsoft.NETCore.App]

You can resolve the problem by installing the specified framework and/or SDK
below is list of installed dotnet sdks

![image](https://user-images.githubusercontent.com/20562096/182303615-49b318ab-1b32-453a-9b46-4513bd74ca20.png)
